jdbc-source/src/main/java/org/springframework/cloud/stream/module/jdbc/source/JdbcSourceProperties.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.jdbc.source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Holds configuration properties for the Jdbc Source module.
+ *
+ * @author Thomas Risberg
+ */
+@ConfigurationProperties
+public class JdbcSourceProperties {
```

Comment:
Right, no reason to validate `maxRowsPerPoll`, because any `< 0` value is valid for the `JdbcPollingChannelAdapter` and we just pick up the full `SELECT` result on the output.